packages/lit-dev-server/package.json

```diff
@@ -17,8 +17,7 @@
     "koa": "^2.13.0",
     "koa-conditional-get": "^3.0.0",
     "koa-etag": "^4.0.0",
-    "koa-static": "^5.0.0",
-    "lit-dev-content": "^0.0.0"
```

Comment:
This dependency is only needed at run time, not at build time, so it does not need to be declared in the dependency graph.

Removing this dependency is helpful because it breaks the cycle:

  server -> content -> tools-esm -> server

Cycles in Lerna are bad because they create undefined build ordering.

In this case, we want to add the content -> tools-esm dependency so that we can run esm tools from the content package's build step. Breaking the cycle by removing server -> content lets us do this.